crates/swc_ecma_parser/src/parser/expr.rs

```diff
@@ -785,6 +785,23 @@ impl<'a, I: Tokens> Parser<I> {
                 }
             }
             return Ok(Box::new(Expr::Arrow(arrow_expr)));
+        } else {
+            // If there's no arrow function, we have to check there's no
+            // AssignProp in lhs to check against assignment in object literals
+            // like (a, {b = 1});
+            for expr_or_spread in paren_items.iter().cloned() {
+                if let PatOrExprOrSpread::ExprOrSpread(e) = expr_or_spread {
+                    if let Expr::Object(o) = *e.expr {
+                        for p in o.props.into_iter() {
```

Comment:
I think `iter()` is enough here.

`Vec::iter` is used to iterate over a reference to vector.

Reply:
As it does not require ownership of the vector, you can remove `cloned()` above.

Reply:
Thanks. I've removed cloned() and i'm accessing through reference.